README.md

```diff
@@ -64,12 +63,12 @@ egrep 'Tests:|tests run:|earl:outcome' reports/* | docker run -i table-reporter
 ```
 The final output should look something like:
 ```sh
-Server              	LDP Basic           	Websockets-pub-sub  	RDF-fixtures
-gold                	0/90                	0/1                 	2/47
-inrupt-pod-server   	0/90                	0/1                 	2/47
-node-solid-server   	15/90               	0/1                 	22/49
-trellis             	47/90               	0/1                 	10/45
-wac-ldp             	57/90               	1/1                 	8/50
+Server              	Websockets-pub-sub  	RDF-fixtures
```

Comment:
Should Websockets-pub-sub be removed until it is actually worked out? Or is there intended to be a particular subset covered?

Reply:
I think it is OK to have there as there is something to test, the actual details are worked out as we go, which is true for everything right now, we're in flux. If we do reach a point where tests are not aligned with the state-of-the-art on specification, then we can consider removing it.